packages/frontend-2/components/viewer/AnchoredPoints.vue

```diff
@@ -86,30 +86,53 @@
       "
     >
       <div
-        class="w-full h-full outline -outline-offset-0 outline-8 rounded-md outline-blue-500/40"
+        class="w-full h-full outline -outline-offset-0 outline-8 rounded-md outline-blue-500"
       >
-        <div class="absolute bottom-4 right-4 p-2 pointer-events-auto">
-          <Portal to="pocket-right">
-            <FormButton
-              v-if="spotlightUserSessionId && spotlightUser"
-              size="xs"
-              class="truncate"
-              @click="() => (spotlightUserSessionId = null)"
-            >
-              <span>Stop Following {{ spotlightUser?.userName.split(' ')[0] }}</span>
-            </FormButton>
+        <div class="absolute top-0 left-[50%] -translate-x-1/2 flex">
```

Comment:
It's not worth changing here, but I'd try and avoid these positioning hacks, trying to use flex where possible. Something like this does the same using only flex
<img width="688" alt="image" src="https://github.com/specklesystems/speckle-server/assets/139135120/536f4a28-7058-46e4-9c9b-4283fb2d5d91">

Reply:
Ah, yes much cleaner. I've been used to centering absolute positioned elements like this. I've updated it.